content/en/developers/dogstatsd/dogstatsd_mapper.md

```diff
@@ -0,0 +1,121 @@
+      tag: 'Documentation'
+      text: 'Official and Community created API and DogStatsD client libraries'
+    - link: 'https://github.com/DataDog/datadog-agent/tree/master/pkg/dogstatsd'
+      tag: 'GitHub'
+      text: 'DogStatsD source code'
+---
+
+With Agent v7.17+, the DogStatsD Mapper feature allows you to convert parts of a metric name submitted to DogStatsD to tags using mapping rules with wildcard and regex patterns.
+
+For example it allows you to transform the metric `airflow.job.duration.<JOB_TYPE>.<JOB_NAME>` into `airflow.job.duration` with two associated tags: `job_type:<JOB_TYPE>` and `job_name:<JOB_NAME>`. To create a mapping rule:
+
+1. [Open your `datadog.yaml` file][1].
+2. Add a [mapping rule configuration block](#mapping-rule) under the `dogstatsd_mapper_profiles` parameter.
+
+## Mapping rule
+
+A mapping rule block have the following layout:
+
+```yaml
+dogstatsd_mapper_profiles:
+    - name: '<PROFILE_NAME>'
+      prefix: '<PROFILE_PREFIX>'
+      mappings:
+          - match: '<METRIC_TO_MATCH>'
+            match_type: '<MATCH_TYPE>'
+            name: '<MAPPED_METRIC_NAME>'
+            tags:
+                '<TAG_KEY>': '<TAG_VALUE_TO_EXPAND>'
+```
+
+With the following placeholders:
+
+| Placeholder             |  Definition                                                                                                                               |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+|  `<PROFILE_NAME>`       | A name to give to your mapping rule profile.                                                                                              |
+| `<PROFILE_PREFIX>`      | The metric name prefix associated to this profile.                                                                                        |
+| `<METRIC_TO_MATCH>`     | The metric name to match. [Wildcard](#wildcard-match-pattern) or [Regex](#regex-match-pattern) are supported.                             |
+| `<MATCH_TYPE>`          | The type of match to apply to the `<METRIC_TO_MATCH>`. Either [`wildcard`](#wildcard-match-pattern) or [`regex`](#regex-match-pattern)    |
+| `<MAPPED_METRIC_NAME>`  | The new metric name to send to Datadog with the tags defined in the same group.                                                           |
+| `<TAG_KEY>`             | The Tag key to associate to the tags collected.                                                                                           |
+| `<TAG_VALUE_TO_EXPAND>` | The tags collected from the `<MATCH_TYPE>` to inline.                                                                                     |
+
+## Wildcard Match Pattern
+
+The wildcard match pattern matches dot-separated metric names using `*` as wildcard. The metric name must follow the [metrics naming convention][2] for the pattern to work.
```

Comment:
Actually, it's a bit more strict than that. Only alphanumeric, `.` and `_` are allowed. 

https://github.com/DataDog/datadog-agent/blob/460ad2c13a7e990ec65660f9dc3aec574c0d6d78/pkg/dogstatsd/mapper/mapper.go#L18